src/jquery.icebearjs.js

```diff
@@ -758,12 +762,34 @@ initJQuery();
 
                 var info = '<div class="category">' + data.caption + '</div>';
 
-                info += "<ul>";
+                if (options.showBullets)
+                {
+                    info += "<ul>";
+                }
+                else
+                {
+                    info += '<ul style="list-style: none;">';
+                }
 
                 for (var i = 0; i < data.content.length; ++i) {
                     var element = data.content[i];
 
-                    info += '<li class="' + element.type + '">' + element.description + '</li>';
+                    if (element.type === "added")
```

Comment:
These if-else blocks can be replaced by using a mapping:

*Far, far away.. somewhere in init..*
```js
textTypes["added"] = options.prependedTextAdded;
textTypes["modified"] = options.prependedTextModified;
textTypes["removed"] = options.prependedTextRemoved;
```
and..
```js
info += '<li class="' + element.type + '">' + textTypes[element.type] + element.description + '</li>';
```